compiler/il/OMRSymbolReference.hpp

```diff
@@ -56,7 +56,7 @@ namespace TR { class ResolvedMethodSymbol; }
 namespace TR { class SymbolReference; }
 template <class T> class TR_Array;
 template <uint32_t> class TR_NodeAliasSetInterface;
-template <uint32_t> class TR_SymAliasSetInterface;
+template <uint32_t> class TR_AliasSetInterface;
```

Comment:
Is it possible to fix the typo in the commit message ("Chnage") please ?

Reply:
Fixed in 2d16c73